src/value.js

```diff
@@ -12,12 +12,28 @@ class Value {
    * @param {*} val - the primitive value from the field cell.
    * @param {string | Field} field - The field from which the value belongs.
    */
-    constructor (val, field) {
-        Object.defineProperty(this, '_value', {
-            enumerable: false,
-            configurable: false,
-            writable: false,
-            value: val
+    constructor (value, rawValue, field) {
+        const numberFormatFn = field.numberFormat && field.numberFormat();
```

Comment:
Keep a default numberFormat method which returns the same value instead of checking if numberFormat is present.